app/actors/hyrax/actors/interpret_visibility_actor.rb

```diff
@@ -227,6 +227,13 @@ def apply_embargo(env, intention)
           return unless env.curation_concern.embargo
           env.curation_concern.embargo.save # see https://github.com/samvera/hydra-head/issues/226
         end
+
+        def reindex_visibility_change(env)
+          return true unless env.curation_concern.visibility_changed?
+          env.curation_concern.update_index
+          # Don't try to handle the indexing response and always return true
+          true
```

Comment:
Should this be a TODO where we handle the indexing response in the future?  If not, can your comment include why it is find to ignore the indexing response?

Reply:
I'm concerned that reaching into the response hash (`response["responseHeader"]["status"]`) will be too brittle but I could put that in.  For what it's worth, it looks like even ActiveFedora ignores the rsolr response: https://github.com/samvera/active_fedora/blob/master/lib/active_fedora/indexing.rb#L74-L79

Reply:
OK